test/FunctionalTests/Microsoft.OData.Core.Tests/ODataContextUriBuilderTests.cs

```diff
@@ -211,13 +211,13 @@ public void EntryContextUriWithSelectString()
 
         [Theory]
         // expand without select, $expand=A
-        [InlineData(ODataVersion.V4, "TestModel.CapitolCity/Districts", "")]
+        [InlineData(ODataVersion.V4, "TestModel.CapitolCity/Districts", "TestModel.CapitolCity/Districts()")]
         [InlineData(ODataVersion.V401, "TestModel.CapitolCity/Districts", "TestModel.CapitolCity/Districts()")]

```

Comment:
Since 4.0 and 4.01 are now the same, we can simplify this test (and ensure 4.0 and 4.01 are the same) by removing the ODataVersion from the InlineData and having the test run for both 4.0 and 4.01.

Reply:
OK. cases merged.

---
In reply to: [224151289](https://github.com/OData/odata.net/pull/1286#discussion_r224151289) [](ancestors = 224151289)